src/Modal/standard-modal.mdx

```diff
@@ -31,7 +31,7 @@ The standard `ModalDialog` composition. `StandardModal` passes all of its props
         footerNode={(
           <ActionRow>
             <p className="small">
-              <Hyperlink href="#">Get help</Hyperlink>
+              <Hyperlink destination="#">Get help</Hyperlink>
```

Comment:
Although this was working, [the `<Hyperlink>` component](https://paragon-openedx.netlify.app/components/hyperlink/) expects `destination` as a prop, not `href`.